package.json

```diff
@@ -104,7 +104,7 @@
     "better-sqlite3": "^8.3.0",
     "compact-encoding": "^2.12.0",
     "corestore": "^6.8.4",
-    "drizzle-orm": "^0.28.1",
+    "drizzle-orm": "0.28.2",
```

Comment:
currently pinned this because type exports are broken in 0.28.3: https://github.com/drizzle-team/drizzle-orm/issues/1088